src/lib.rs

```diff
@@ -107,4 +118,24 @@ mod tests {
         assert_eq!(bank.rdh_version, Some("300".to_string()));
         assert_eq!(bank.pin_tan_version, Some("300".to_string()));
     }
+
+    // These tests use the real CSV input data.
+    // As such, the tests might break randomly if the input document changes.
+    // This should happen rarely enough that it's still worthwhile to test with real data, though.
+
+    #[test]
+    fn getting_random_bank_works() {
+        let bank_code = "44761312";
+        let bank = get_bank_by_bank_code(bank_code).unwrap();
+
+        assert_bank_matches(&bank);
+    }
+
+    #[test]
+    fn getting_bank_by_bic() {
```

Comment:
Could you rename this to `get_bank_by_bic`?

Reply:
same as above.